app/UpdateStore.tsx

```diff
@@ -74,6 +75,7 @@ export function UpdateStore(props: {
             if (suburb === "") {
               return setErrorMessage("Suburb cannot be empty");
             }
+            updateStore(store, displayName, fullName, suburb);
             onUpdateStoreSuccess();
```

Comment:
I feel like if you don't `await` the `updateStore` function, then `onUpdateStoreSuccess` will start running before `updateStore` is done

Reply:
https://github.com/auberginearms/groceryDatabaseApp/pull/21/commits/ffc22f561fa5d85df98e5cb6e584c6f6dd5db484